Index: Monsoon/MainWindow.cs
===================================================================
--- Monsoon/MainWindow.cs	(Revision 148377)
+++ Monsoon/MainWindow.cs	(Arbeitskopie)
@@ -854,8 +854,7 @@
 			// This should be stored before the torrent is stopped
 			StoreTorrentSettings ();
 			
-			foreach (WaitHandle h in this.torrentController.Engine.StopAll())
-				h.WaitOne (TimeSpan.FromSeconds(2), false);	
+			this.torrentController.Engine.StopAll();
 
 //			List<WaitHandle> handles = new List<WaitHandle> ();
 //			foreach (Download manager in torrents.Keys){
